src/isar/config/settings.py

```diff
@@ -183,8 +183,11 @@ class Settings(BaseSettings):
     # Type of robot ISAR is monitoring
     ROBOT_TYPE: str = Field(default="robot")
 
-    # Name or unique ID of robot
-    ROBOT_ID: str = Field(default="R2-D2")
+    # Name of robot
+    ROBOT_NAME: str = Field(default="R2-D2")
```

Comment:
Can we change the default robot name to something a little more clear is the default like
"default"?

Reply:
`Placebot`

Reply:
done